server/main.go

```diff
@@ -1,27 +1,32 @@
 package main
 
 import (
-	"database/sql"
-	_ "github.com/mattn/go-sqlite3"
 	"log"
+
+	"github.com/UCSC-CSE123/beavertail/server/models"
+	"github.com/UCSC-CSE123/beavertail/server/service/datagram"
+	_ "github.com/mattn/go-sqlite3"
 )
 
 func main() {
-	db, err := sql.Open("sqlite3", "./busdata.db")
+	// Establish a connection to the database.
+	db, err := models.NewDB("./busdata.db")
```

Comment:
Can you make the database path configurable by argument?

Reply:
Paths are now figurable via flags or a yaml file (: